pkg/security/secl/compiler/eval/operators.go

```diff
@@ -579,3 +579,244 @@ func ArrayBoolContains(a *BoolEvaluator, b *BoolArrayEvaluator, opts *Opts, stat
 		isDeterministic: isDc,
 	}, nil
 }
+
+// CIDREquals evaluates CIDR ranges
+func CIDREquals(a *CIDREvaluator, b *CIDREvaluator, opts *Opts, state *State) (*BoolEvaluator, error) {
+	isDc := isArithmDeterministic(a, b, state)
+
+	arrayOp := func(as IPMatcher, bs IPMatcher) bool {
```

Comment:
I know that because of a copy/past but just `op` instead of `arrayOp` as this not for an array here